ui_tests/src/test/resources/com/saucedemo/account/signinNegative.feature

```diff
@@ -0,0 +1,52 @@
+Feature: Sign In
+
+
+  @signIn
+  Scenario: Sign in with invalid password
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set 'thisIsSomeInvalidPassword' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+
+  @signIn
+  Scenario: Sign in with invalid user name
+    Given I open start page
+    When I set 'thisIsSomeInvalidUser' as username on Sign in page
+    And I set 'standard_password' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  @signIn
+  Scenario: Sign in with empty user and password
+    Given I open start page
+    When I set ' ' as username on Sign in page
+    And I set ' ' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  Scenario: Sign in with valid user name and empty password
+    Given I open start page
+    When I set 'standard_user' as username on Sign in page
+    And I set ' ' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  Scenario: Sign in with empty user name and valid password
+    Given I open start page
+    When I set ' ' as username on Sign in page
+    And I set 'secret_sauce' as password on Sign in page
+    And I click 'Sign In' button on Sign in page expecting failure
+    Then Error message 'Epic sadface: Username and password do not match any user in this service' is available on Sign in page
+
+  Scenario Outline: Sign in
+    When I set '<username>' as username on Sign in page
+    And I set '<password>' as password on Sign in page
+    And I click 'Sign In' button on Sign in page
```

Comment:
There're two step definitions that allow you to click on sign in button:
1. `I click 'Sign In' button on Sign in page` it automatically (I'll show how it done later on) verifies that we are redirected to Products page, i.e. we are singed in
2. `I click 'Sign In' button on Sign in page expecting failure` it automatically verifies that we are remaining on the Sign In page, i.e. we are not signed it
Not the appropriate step is used here